README.md

```diff
@@ -72,3 +72,9 @@ $ python -m unittest discover tests/
 When using **signac** as part of your work towards a publication, we would really appreciate that you acknowledge **signac** appropriately.
 We have prepared examples on how to do that [here](http://docs.signac.io/en/latest/acknowledge.html).
 **Thank you very much!**
+
+## NumFOCUS

```

Comment:
I don't like that this is its own second-level heading - I think it fits alright under Acknowledgements (if the heading were removed), though the intention is different (conveying "orgs we acknowledge" instead of "how to acknowledge signac").

I think I might prefer to have only the badge in the README and then put the larger logo in the footer of our website. Or we could just remove the `## NumFOCUS` heading and logo, leaving the line of text with a link in Acknowledgements. Whatever you think is fine, I'm certainly happy to represent our NumFOCUS affiliation.

Reply:
I think I'd prefer to remove the heading and logo and have the big logo only on the website.